### DISCUSSION TITLE
Can't add "USING SCOPE" clause to method chain queries

### DISCUSSION THREAD

**Discussion by ghost** [jsforce/jsforce#1480] [Ideas]
Hi,
after some research, I can't find any way to add a `USING SCOPE` clause to my query build using method chaining.

What I'm trying to do is replicating this query
```TypeScript
const result = await connection
    .query('SELECT Id, Name FROM Account USING SCOPE my_territory LIMIT 10');
```

but using method chaining instead
```TypeScript
const result = await connection
    .sobject('Account')
    // Miss some method to add "USING SCOPE my_territory"
    .find()
    .select(['Id', 'Name'])
    .limit(10)
```

Is there any way I can avoid writing raw SOQL for that?

Thank you

**Comment by paustint** [comment]
One alternative would be to use https://github.com/paustint/soql-parser-js to compose your queries.
